shell/platform/darwin/ios/BUILD.gn

```diff
@@ -270,6 +270,7 @@ shared_library("ios_test_flutter") {
     "//build/config:symbol_visibility_hidden",
   ]
   sources = [
+    "../../../../third_party/spring_animation/SpringAnimationTest.mm",
```

Comment:
I think tests from 3rd party library should be added in its own BUILD file. I'm not sure if our CI runs all the tests in the 3rd party folders tho, maybe someone from the engine team could explain or provide an example? @chinmaygarde @dnfield

Reply:
> I'm not sure if our CI runs all the tests in the 3rd party folders tho

Yes. `testing/run_tests.py --type=objc` will run the `SpringAnimationTest.mm`

Reply:
Added. Plz re-review^_^

You can see new tests in  `Mac Unopt`